nbbuild/antsrc/org/netbeans/nbbuild/extlibs/DownloadBinaries.java

```diff
@@ -161,6 +169,13 @@ public void execute() throws BuildException {
                             } else {
                                 success &= fillInFile(hashAndFile[0], hashAndFile[1], manifest, () -> legacyDownload(hashAndFile[0] + "-" + hashAndFile[1]));
                             }
+                            if (report != null) {
+                                Files.write(
+                                        report.toPath(),
+                                        (hashAndFile[0] + ";" + hashAndFile[1] + ";" + include + "\n").getBytes(),
+                                        StandardOpenOption.APPEND);
```

Comment:
The append looks strange here (you have a delete / touch before this). Why not just use `StandardOpenOption.CREATE`?

Reply:
This would lead to having only last binary in the file.